hudi-spark-datasource/hudi-spark/src/test/java/org/apache/hudi/functional/TestDataSkippingWithMORColstats.java

```diff
@@ -318,6 +318,7 @@ private Map<String, String> getOptions() {
     options.put(HoodieMetadataConfig.ENABLE.key(), "true");
     options.put(HoodieMetadataConfig.ENABLE_METADATA_INDEX_COLUMN_STATS.key(), "true");
     options.put(HoodieMetadataConfig.COLUMN_STATS_INDEX_FOR_COLUMNS.key(), "trip_type");
+    options.put(HoodieMetadataConfig.ENABLE_METADATA_INDEX_PARTITION_STATS.key(), "false");
```

Comment:
this class has some unit tests that deliberately corrupt parquet file by replacing a valid file with file of 0 bytes. In that case, keeping partition stats enabled throws error. To avoid that, I had disabled. In the latest commit, I have disabled only for tests that simulate corrupt parquet file as mentioned above, and not the whole test suite.